pkg/component/dependencywatchdog/access.go

```diff
@@ -37,10 +43,19 @@ const (
 	DefaultProbeInterval = 30 * time.Second
 	// DefaultWatchDuration is the default value of the total duration for which a DWD Weeder watches for any dependant Pod to transition to CrashLoopBackoff after the target service has recovered.
 	DefaultWatchDuration = 5 * time.Minute
+	// KubeConfigSecretName is the name of the kubecfg secret with internal DNS for external access.
+	KubeConfigSecretName = gardenerutils.SecretNamePrefixShootAccess + "dependency-watchdog-probe"
+	// managedResourceTargetName is the name of the managed resource created for DWD.
+	managedResourceTargetName = "shoot-core-dependency-watchdog"
+	// DefaultKCMNodeMonitorGraceDuration is the default value for the NodeMonitorGraceDuration parameter of KCM.
+	DefaultKCMNodeMonitorGraceDuration = 40 * time.Second
```

Comment:
I guess this constant rather belongs to the `kubecontrollermanager` package, doesn't it?